### DISCUSSION TITLE
Equipped items shouldn't use Inventory Slots.

### DISCUSSION THREAD

**Discussion by Arufonsu** [AscensionGameDev/Intersect-Engine#1493] [Ideas]
So here's my first enchancement/feature request...

### **Is your feature request related to a problem? Please describe.**
Equipped items shouldn't use up Inventory Slots... since you are... you know, wearing the gears, they should only use up the inventory slots while unequipped. This is a common behaviour in most of the MMOs out there, and its also the most logic thing to do.. Why would a hat stay in my item inventory while i'm currently wearing it? is it stored in two places at the same time?

### **Describe the solution you'd like**
Like in common old school MMOs (Tibia/RO/Almost any MMO) we need an **Equipment Inventory** and an **Item Inventory**.
Example of common MMOs: You double click an equipment in order to wear it, and it moves from your **item inventory** into your **equipment inventory**, freeing up the slot in the item inventory, thus using the equipment inventory slot corresponding to the equipment type.

### **Additional context**
This would be useful to, lets say, save item inventory slots, create Eyes/Hair/Cosmetics paperdolls that can be given/equipped and locked, it would enchance the QoL regardless organization, in fact, it would be way easier to tell which item you are currently wearing and which not, instead of an equipment label (the way its currently being handled).

**Comment by jcsnider** [comment]
Open to seeing a PR for this if anyone wants to take a stab at it.
